src/householder/householder.service.ts

```diff
@@ -1,12 +1,19 @@
 import { Injectable } from '@nestjs/common';
 
+import { PrismaService } from '../prisma/prisma.service';
+import type { CreateHouseholderInput } from './dto/input/create-householder.input';
+
 @Injectable()
 export class HouseholderService {
-  findAll() {
-    return `This action returns all householder`;
-  }
+  constructor(private readonly prisma: PrismaService) {}
 
-  findOne(id: number) {
-    return `This action returns a #${id} householder`;
+  async create(data: CreateHouseholderInput) {
+    try {
+      const householder = await this.prisma.householder.create({ data });
+      return householder;
+    } catch (error) {
+      console.log({ error });
```

Comment:
it's a good practice to use Logger class instead of console. because in future you can change the place of logs with minimum config.

```javascript 
logger.error({ errorOnCreateHouseholder: error });
```